src/main/java/net/openhft/chronicle/queue/ChronicleReaderMain.java

```diff
@@ -136,6 +137,9 @@ protected void configureReader(final ChronicleReader chronicleReader, final Comm
         if (commandLine.hasOption('s')) {
             chronicleReader.suppressDisplayIndex();
         }
+        if (commandLine.hasOption('z')) {
+            System.setProperty("mtlc.zoneId", ZoneId.systemDefault().toString());
+        }
```

Comment:
I wanted to leave this in ChronicleReaderMain not in ChroniclReader because its a global change